### PR TITLE
added_avatar

### PR DESCRIPTION
**Pull request by dubeyaditya29**
Have added three avatar of variable sizes (small, medium and large) 

https://user-images.githubusercontent.com/48158220/152098842-935e1584-c50a-4714-ad0d-4bb1c6f43400.mp4

### CODE REVIEW

File: components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  gap: 1rem;
+}
+xmp {
+  border: 2px solid var(--color-grey);
+  padding: 1rem;
+  width: 80%;
+  color: var(--color-royalBlue);
+}
+.avatar {
+  height: 70px;
```
Comment (shivampandey0) [inline components/Avatar/avatar.css]: can use rems for larger than 10 px units

File: components/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 2rem;
+  padding: 2rem;
+}
+.content {
+  display: flex;
```
Comment (shivampandey0) [inline components/Avatar/avatar.css:13]: container, content, leftNav .... use similar properties of flex. can optimize it

**Review by shivampandey0 — COMMENTED** [review]
added some suggestions

File: components/Avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+xmp {
+  border: 2px solid var(--color-grey);
+  padding: 1rem;
+  width: 80%;
+  color: var(--color-royalBlue);
+}
+.avatar {
+  height: 70px;
+  width: 70px;
+  border-radius: 50%;
```
Comment (lannisterrr) [inline components/Avatar/avatar.css:40]: can make seperate variable for this.

File: components/Avatar/avatar.css (hunk 2 above)
Comment (dubeyaditya29) [reply components/Avatar/avatar.css:13]: have removed the flex property from leftNav.

File: components/Avatar/avatar.css (hunk 1 above)
Comment (dubeyaditya29) [reply components/Avatar/avatar.css]: yes have changed to their equivalent rem values.

File: components/Avatar/avatar.css (hunk 3 above)
Comment (dubeyaditya29) [reply components/Avatar/avatar.css:40]: Have created separate variables for sizes also as they are >10px so changed them to their relevant rem values.

File: components/Avatar/avatar.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+          modifier class to reset the border-radius to a more appropriate level.
+        </p>
+
+        <img
+          class="avatar-small"
+          src="../../assets/my-img.jpg"
+          alt="my-profile"
+        />
+
+        <xmp>
```
Comment (shivampandey0) [inline components/Avatar/avatar.html]: Looks like xmp is deprecated.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/xmp

File: components/Avatar/avatar.css (hunk 2 above)
Comment (shivampandey0) [reply components/Avatar/avatar.css:13]: don't leave unused code as commented, remove it.

**Review by shivampandey0 — COMMENTED** [review]
LGTM

File: components/Avatar/avatar.html (hunk 4 above)
Comment (dubeyaditya29) [reply components/Avatar/avatar.html]: removed xmp and used pre tag thanks for the link.

**Review by piyushsh25 — COMMENTED** [review]
The code looks good to me. 👍👍

**Pull request merged**